command/shell.go

```diff
@@ -153,19 +153,21 @@ func (c *shellExec) Execute(ctx context.Context,
 		return err
 	}
 
-	logger.System().Debugf("spawned shell process with pid %d", localCmd.GetPid())
+	pid := localCmd.GetPid()
+
+	logger.System().Debugf("spawned shell process with pid %d", pid)
 
 	// Call the platform's process-tracking function. On some OSes this will be a noop,
 	// on others this may need to do some additional work to track the process so that
 	// it can be cleaned up later.
-	subprocess.TrackProcess(conf.Task.Id, localCmd.GetPid(), logger.System())
+	subprocess.TrackProcess(conf.Task.Id, pid, logger.System())
```

Comment:
command/shell.go:163:27: too many arguments in call to logger.Execution().Debug
	have (string, int)
	want (interface {})
command/shell.go:167:19: too many arguments in call to "github.com/evergreen-ci/evergreen/vendor/github.com/pkg/errors".Wrap
	have (error, string, int)
	want (error, string)